services/app/src/components/chessBoard/ChessBoard.jsx

```diff
@@ -1,54 +1,41 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useContext } from 'react';
+import { ChessBoardContext } from '../../contexts/ChessBoardContext';
+
 import '../../static/style/chess-board.css';
 
-const ChessBoard = ({ numberOfRows, numberOfColumns }) => {
-  const alphabetArray = [...'abcdefghijklmnopqrstuvwxyz'];
-  const board = [...Array(numberOfRows + 1).keys()].map((_row, rowIndex) =>
-    [...Array(numberOfColumns + 1).keys()].map((_tile, tileIndex) => ({
-      attribute:
-        tileIndex === 0 && rowIndex !== numberOfRows
-          ? numberOfColumns - rowIndex
-          : tileIndex !== 0 && rowIndex === numberOfRows
-          ? alphabetArray[tileIndex - 1]
-          : null,
-      isFilled:
-        tileIndex !== 0 && rowIndex !== numberOfRows
-          ? (rowIndex % 2 && !(tileIndex % 2)) ||
-            (!(rowIndex % 2) && tileIndex % 2)
-          : null,
-    })),
-  );
+const ChessBoard = () => {
+  const { placeItemOnBoard, board } = useContext(ChessBoardContext)
+
+  const displayBoard = (
+     board.map((row, rowIndex) => (
+       <tr key={rowIndex}>
+         {row.map((tile, tileIndex) => (
+           <td
```

Comment:
Manau reiketu `Tile` iskelti i atkira komponenta ir padaryti taip
```
const ChessBoard = () => {
  const { placeItemOnBoard, board } = useContext(ChessBoardContext)

  return (
    <table id="chess-board">
      {board.map((row, rowIndex) => (
        <tr key={rowIndex}>
          {row.map((tile, columnIndex) => (
            <Tile
              key={tileIndex}
              attribute={tile.attribute}
              items={tile.items}
              onClick={placeItemOnBoard}
              row={rowIndex}
              column={columnIndex}
            />
          ))}
        </tr>
      ))}
    </table>
  );
};
```

Kur `Tile.jsx`:
```
const Tile = ({ attribute, items, onClick, row, column }) => {
  const someSensibleNameThatDescribesWhatThisConditionMeans = () => {
    return !tile.attribute && row !== board.length - 1 && column !== 0
  }

  const _onClick = useCallback(() => {
    onClick(row, column)
  }, [onClick]);

  return (
    <td
      onClick={_onClick}
      role='presentation'
      className={
        classNames({
          'black-tile': tile.isFilled,
          'white-tile': someSensibleNameThatDescribesWhatThisConditionMeans(),
        })
      }
    >
      {attribute || items}
    </td>
  )
}
```

cia classNames yra package'as, kuris is true/false objekto sulipdo klasiu vardus - https://www.npmjs.com/search?q=classNames